README.md

```diff
@@ -1,2 +1,46 @@
 # av_ouster
-This repository hosts the required enviorment to run the ROS 2 Ouster sensor driver.
+This repository hosts the required environment for running the ROS 2 Ouster LIDAR driver, specifically configured for running with the [OS2-128](https://ouster.com/products/hardware/os2-lidar-sensor) sensor.
+
+
+## Docker container
+- **Operating System**: Ubuntu 22.04
+- **ROS Distribution**: Humble Hawksbill
+- **Velodyne Drivers**:
+    - `ros-humble-ros2-ouster` ( version `0.4.2-1`): Manages communication with the OS2  lidar
```

Comment:
Should we be writing the driver version in the readme even though we haven't pinned it in the Dockerfile? This will go out of date and the Readme will not reflect it

Reply:
I guess what I wanted is to specify what version driver we "know" was installed and tested more than the one is going to stay forever as we are not pinning the versions to avoid conflicts. But, maybe we can add a small note that mentions:
" Although the driver versions are not pinned in the Dockerfile, version <c.y.u> is the latest release available via apt and has been tested on the vehicle."

If we do not think is worth it, we can just remove the version string

Or something like that

Reply:
I do like your suggestion to add the "last known working version", that could help in future debugging. Plz add!

Reply:
Done